highcharts-api/highcharts/14-networkgraph-collapse/main.js

```diff
@@ -1,107 +1,103 @@
- //source for the function below: https://jsfiddle.net/BlackLabel/zrL8w067/
- const setPrototypeMouseOut = (HC) => HC.Series.prototype.onMouseOut = () => HC.fireEvent(this, 'mouseOut');
- setPrototypeMouseOut(Highcharts);
- 
- //some basic recursion
- const hideLink = (link) => {
-   link.graphic.hide();
-   link.toNode.graphic.hide();
-   link.toNode.dataLabel.hide();
-   link.toNode.linksFrom.forEach(childLink => hideLink(childLink));
- };
- const showLink = (link) => {
-   link.graphic.show();
-   link.toNode.graphic.show();
-   link.toNode.dataLabel.show();
-   link.toNode.linksFrom.forEach(childLink => showLink(childLink));
- };
+Highcharts.Series.prototype.onMouseOut = () => HC.fireEvent(this, 'mouseOut');
```

Comment:
```suggestion
Highcharts.Series.prototype.onMouseOut = () => Highcharts.fireEvent(this, 'mouseOut');
```

Shouldn't it be like this?

Reply:
Oh wow, sorry about that, I will fix after lunch!